server/core/region.go

```diff
@@ -229,6 +229,17 @@ func (r *RegionInfo) NeedMerge(mergeSize int64, mergeKeys int64) bool {
 	return r.GetApproximateSize() <= mergeSize && r.GetApproximateKeys() <= mergeKeys
 }
 
+// IsOversized indicates whether the region is oversized.
+func (r *RegionInfo) IsOversized(maxSize int64, maxKeys int64) bool {
+	return r.GetApproximateSize() >= maxSize || r.GetApproximateKeys() >= maxKeys
+}
+
+// IsUndersized indicates whether the region is undersized.
+// PS: This is an alias of `NeedMerge`.
+func (r *RegionInfo) IsUndersized(minSize int64, minKeys int64) bool {
```

Comment:
why not rename  `NeedMerge`

Reply:
In the `MergeChecker`, `NeedMerge` is a more clear name. So I want to keep both to use in different contexts.